src/app/components/ATIAnalytics/canonical/sendBeaconOperaMiniScript/index.ts

```diff
@@ -7,10 +7,7 @@ const sendBeaconOperaMiniScript = (atiPageViewUrlString: string) => `
       var atiPageViewUrl = "${atiPageViewUrlString}";
       atiPageViewUrl += document.referrer ? "&ref=" + document.referrer : '';
 
-      var xhr = new XMLHttpRequest();
-      xhr.open("GET", atiPageViewUrl, true);
-      xhr.withCredentials = true;
-      xhr.send();
+      window.sendStaticBeacon && window.sendStaticBeacon(atiPageViewUrl)
```

Comment:
Did we need to do this null check?

```suggestion
      window.sendStaticBeacon(atiPageViewUrl)
```

Reply:
```suggestion
      window.sendStaticBeacon && window.sendStaticBeacon(atiPageViewUrl);
```

Reply:
Removed it - it's not really needed.